api/fixtures/example.go

```diff
@@ -416,6 +416,10 @@ web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
 			},
 		}
 		nodePools = append(nodePools, nodePool)
+	case hyperv1.NonePlatform, hyperv1.AgentPlatform:
```

Comment:
Why does a nodePool is created for NonePlatform?

Reply:
In case the user specify node-pool-replicas bigger than -1, the NodePool should be created
Probably the user change manually the value in order to have the ign service created (default value - NodePool is not created)
That was the logic before

Reply:
we need to consider IBMCloud in this as well